project/UI/Playground/Playground.swift

```diff
@@ -54,35 +58,34 @@ public struct PlaygroundCommand: ParsableCommand {
                     name: self.name,
                     output: self.output.url,
                     platform: self.platform,
-                    dependencies: self.dependencies)
+                    dependencies: self.dependencies ?? .bow(.version()))
                 },
                 { arg in self.nefPlayground(
                     xcodePlayground: arg.url,
                     name: self.name,
                     output: self.output.url,
                     platform: self.platform,
-                    dependencies: self.dependencies)
+                    dependencies: self.dependencies ?? .spm)
                 })
     }
     
     // MARK: attributes
-    private var dependencies: PlaygroundDependencies {
-        let dependencies: PlaygroundDependencies
+    private var dependencies: PlaygroundDependencies? {
         if let version = bowVersion {
-            dependencies = .bow(.version(version))
+            return .bow(.version(version))
         } else if let branch = bowBranch {
-            dependencies = .bow(.branch(branch))
+            return .bow(.branch(branch))
         } else if let commit = bowCommit {
-            dependencies = .bow(.commit(commit))
+            return .bow(.commit(commit))
         } else if let podfileURL = podfile?.url {
-            dependencies = .podfile(podfileURL)
+            return .podfile(podfileURL)
         } else if let cartfileURL = cartfile?.url {
-            dependencies = .cartfile(cartfileURL)
+            return .cartfile(cartfileURL)
+        } else if spm {
+            return .spm
         } else {
-            dependencies = .bow(.version(""))
+            return nil
```

Comment:
I think this modeling needs to be revisited. This allows an ambiguous input from the user, where you can supply both a Podfile and a Cartfile, and only knowing what is the priority resolving them explains the observed outcome. Also, what happens if I want to use a specific Bow branch, but with SPM?

Reply:
Right now, three dependency managers are mutually exclusive. If you set more than one, the behaviour will be not expected because it is not supported; so the order at this point is not important. Following your doubt, I have just thought we could detect if a user set more than one dependency manager and throw a `fatal-error`

Reply:
done!